### PR TITLE
Milestone2 todo list

### PR DESCRIPTION
**Pull request by elisha2kyakpo1**
In this Milestone, I have implemented the following:

1. Add 2 new JavaScript files and import them as modules:
2. Implemented the drag and drop functionality.
3. Add event listeners to list items (dragstart, dragover, dropend).
4. Add functionality to add to local storage.

### CODE REVIEW

File: src/drag.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    });
+
+    lists.addEventListener('dragend', () => {
+      setTimeout(() => lists.classList.remove('draggable-start'), 0);
+    });
+  });
+
+  ul.addEventListener('dragover', (e) => {
+    e.preventDefault();
+    dragAfterElement(ul, e.clientY);
```
Comment (Dipeshtwis) [inline src/drag.js]: - [ ] After drag and drop and updating the status, the updated array of items should stored in local storage, so the user gets the correct list values after the page reloads. Currently it's not updating and showing me the initial value. Make sure to reload the page to test the array is stored. :slightly_smiling_face:

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  list-style-type: none;
+  padding: 10px;
+  text-align: center;
+  border-bottom: 1px solid #888;
+}
+
+.list-item.draggable-start {
+  opacity: 0.5;
+}
+
```
Comment (Dipeshtwis) [inline src/style.css]: - [ ] Also make sure keep the UI similar.

### Your design 

Screenshot from 2021-07-16 23-21-29

### Suggested design (also shown [in this video](https://www.youtube.com/watch?v=AcUd-_Yjjqg))

Screenshot from 2021-07-16 23-23-15

Please note that the video have more features than the current project, you don't need to style all the features not implemented yet in the video.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @elisha2kyakpo1, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Contain methods related to drag / drop and status updates functionality..

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [ ] README file should have steps about how to clone and run the project in local. You can use microverse suggested [readme template](https://github.com/microverseinc/readme-template/blob/master/README.md).

**_Also check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi  @elisha2kyakpo1, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**